gsl_notification/tests/test_views.py

```diff
@@ -691,3 +694,80 @@ def test_create_arrete_views(correct_perimetre_client_with_user_logged):
     assert modele_en_base
     assert modele_en_base.created_by == correct_perimetre_client_with_user_logged.user
     assert modele_en_base.logo_alt_text == "Texte alternatif du logo"
+
+
+### delete-modele-arrete
+
+
+def test_delete_modele_arrete_with_correct_perimetre():
+    departement_perimetre = PerimetreDepartementalFactory()
+    user = CollegueFactory(perimetre=departement_perimetre)
+    client = ClientWithLoggedUserFactory(user)
+    modele_arrete = ModeleArreteFactory(
+        perimetre=departement_perimetre, name="Mon modèle"
+    )
+    url = reverse("gsl_notification:delete-modele-arrete", args=[modele_arrete.id])
+
+    response = client.post(url)
+
+    expected_redirect_url = reverse(
+        "gsl_notification:modele-arrete-liste", args=[modele_arrete.dotation]
+    )
+    assert response.status_code == 302
+    assert response.url == expected_redirect_url
+
+    assert ModeleArrete.objects.count() == 0
+    messages = get_messages(response.wsgi_request)
+    assert len(messages) == 1
+    message = list(messages)[0]
+    assert message.level == 20
+    assert message.message == "Le modèle d’arrêté “Mon modèle” a été supprimé"
```

Comment:
Oui bonjour c'est le comité de la ponctuation.

```suggestion
    assert message.message == "Le modèle d’arrêté “Mon modèle” a été supprimé."
```